tap_paycor/__init__.py

```diff
@@ -52,6 +52,32 @@ def discover():
     return Catalog(streams)
 
 
+def refresh_token(args):
+    root_url = 'https://apis.paycor.com'
+    config = args.config
+    key = os.environ['PAYCOR_API_SUBSCRIPTION_KEY']
+    url = f"{root_url}/sts/v1/common/token?subscription-key={key}"
+    headers = { "content-type": "application/x-www-form-urlencoded" }
+    data = {
+        "grant_type": "refresh_token",
+        "refresh_token": os.environ["PAYCOR_REFRESH_TOKEN"],
```

Comment:
@chriscauley singer taps are supposed to take all their input config from config.json rather than environment, so with this code moved inside the tap we should be taking all existing values from the incoming config

It will need to be the Airflow tasks concern to either build config.json from a mix of env data and secret manager data, or we might as well just shove the entire config.json into a secret manager value

Reply:
Fixed in [3830e32](https://github.com/JarvusInnovations/tap-paycor/pull/7/commits/3830e322da3ebdb63825ca5b975fb47bb9c8389e). I'll update the other repo now.